art/attacks/evasion/auto_projected_gradient_descent.py

```diff
@@ -445,8 +523,12 @@ def generate(self, x: np.ndarray, y: Optional[np.ndarray] = None, **kwargs) -> n
 
                 var_w = [math.ceil(p * self.max_iter) for p in var_w]
 
-                eta = self.eps_step
-                self.count_condition_1 = 0
+                # eta = self.eps_step
+                # self.count_condition_1 = 0
+                # modification for image-wise stepsize update
+                bs = x_k.shape[0]
```

Comment:
Would it be possible to change the name `bs` into a name with at least 3 characters?